client/styles/components/_console.scss

```diff
@@ -18,28 +18,53 @@
 	}
 
 	// assign styles to different types of console messages
-	.preview-console__log {
-		@include themify(){
-			color: getThemifyVariable('secondary-text-color');
-		}
+	.preview-console__log, .preview-console__info{
```

Comment:
Since now all of the console messages have the same class in common (`.preview-console__message`), you don't have to repeat these styles a bunch of times. You can do
```SCSS
.preview-console__message {
  flex: 1 0 auto;
  position: relative;
}
```